src/transformers/models/patchtst/configuration_patchtst.py

```diff
@@ -0,0 +1,274 @@
+        norm_eps (`float`, *optional*, defaults to 1e-05):
+            A value added to the denominator for numerical stability of normalization.
+        attention_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for the attention probabilities.
+        dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability for all fully connected layers in the encoder, and decoder.
+        positional_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability in the positional embedding layer.
+        dropout_path (`float`, *optional*, defaults to 0.0):
+            The dropout path in the residual block.
+        ff_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability used between the two layers of the feed-forward networks.
+        bias (`bool`, *optional*, defaults to `True`):
+            Consider bias in the feed-forward networks.
+        activation_function (`str`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (string) in the encoder.`"gelu"` and `"relu"` are supported.
+        pre_norm (`bool`, *optional*, defaults to `True`):
+            Normalization is applied before self-attention if pre_norm is set to `True`. Otherwise, normalization is
+            applied after residual block.
+        positional_encoding_type (`str`, *optional*, defaults to `"sincos"`):
+            Positional encodings. `"zeros"`, `"normal"`, `"uniform"' and `"sincos"` are supported.
+        learn_pe (`bool`, *optional*, defaults to `False`):
+            Whether the positional encoding is updated during training.
+        use_cls_token (`bool`, *optional*, defaults to `False`):
+            Whether cls token is used.
+        init_std (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated normal weight initialization distribution.
+        shared_projection (`bool`, *optional*, defaults to `True`):
+            Sharing the projection layer across different channels in the forecast head.
+        seed_number (`Optional`, *optional*):
+            Seed number used for random masking. If unset, no seed is set.
+        scaling (`Union`, *optional*, defaults to `"mean"`):
+            Whether to scale the input targets via "mean" scaler, "std" scaler or no scaler if `None`. If `True`, the
+            scaler is set to "mean".
+        mask_input (`bool`, *optional*, defaults to `False`):
+            Apply masking during the pretraining.
+        mask_type (`str`, *optional*, defaults to `"random"`):
+            Masking type. Only `"random"` and `"forecast"` are currently supported.
+        random_mask_ratio (`float`, *optional*, defaults to 0.5):
+            Masking ratio is applied to mask the input data during random pretraining.
+        forecast_mask_patches (`List`, *optional*, defaults to `[2, 3]`):
```

Comment:
Otherwise reads as: should I forecast the mask patches?

```suggestion
        forecast_mask_patch_lengths (`List`, *optional*, defaults to `[2, 3]`):
```